common/src/main/java/com/scottlogic/deg/common/profile/HelixDateTime.java

```diff
@@ -0,0 +1,71 @@
+package com.scottlogic.deg.common.profile;
+
+import com.scottlogic.deg.common.ValidationException;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
+import java.time.temporal.ChronoField;
+import java.time.temporal.TemporalAccessor;
+
```

Comment:
I think this class and the similar classes (HelixNumber etc) need some documentation.  You've explained in person why they're being used but I don't think it'll be obvious for anyone reading the code.

If they're all just used to provide validation I wonder if we can just use a validation method and avoid creating these classes, it'll depend on how many places they're created I suppose.

Reply:
also if the purpose of these classes is generator validation, then they don't really belong in the common, as output doesn't care about generator validation.

this also makes some thing awkward like, validating inputted custom generator scripts, as that really doesn't seem right to be common.

Reply:
Agreed, this has all moved to generator in #1461. The future of these classes may be short lived and so its worth coming back to this separately.